Peekabook/Peekabook/Global/Literals/StringLiterals.swift

```diff
@@ -24,4 +24,8 @@ struct I18N {
         static let cancel = "취소하기"
         static let recommend = "추천하기"
     }
+    
+    struct PlaceHolder {
```

Comment:
해당 쓰이는 뷰를 적는게 더 좋아보입니당
```suggestion
    ~VC {
````